### PR TITLE
Feature/load credentials from env

### PR DESCRIPTION
**Pull request by PhilippSpo**
@emmenko @hajoeichler @svenmueller 
Should we add some kind of priority to variables loaded from env? So they get taken automatically if present (no matter what other sources for credentials contain).

### CODE REVIEW

**Comment by emmenko** [comment]
Did you have a look here?

https://github.com/sphereio/sphere-node-cli/blob/master/lib/utils/load-credentials.js

**Comment by PhilippSpo** [comment]
:+1:  That's nice, but it is not in the credentials utils sadly. Should I adapt either of the to codes to use the same env var names? If so which one? @svenmueller suggested to use `CTP` as a prefix since the product is called commerce tools platform and not sphere.

**Comment by svenmueller** [comment]
then i would rather vote for consistency and keep using the `sphere` prefix

**Comment by PhilippSpo** [comment]
@svenmueller @emmenko renamed the env variables and adapted code to be equivalent to what @emmenko did in the `sphere-node-cli`

**Comment by hajoeichler** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -211,6 +213,17 @@ By default the module will try to read the credentials from the following locati
...
 
+##### From environment variables
+
+This is a little bit more restricted, since you can only define one set of credentials with the environment variables. Nevertheless this is very useful for deployments, where you really only need one set of credentials per deployment.
+You can define your credentials using these variables:
+
+```sh
+export CTP_PROJECT_KEY="your-project-key"
+export CTP_CLIENT_ID="your-client-id"
+export CTP_CLIENT_SECRET="your-client-secret"
```
Comment (emmenko) [inline README.md]: Wrong names

**Comment by emmenko** [comment]
:+1:

**Pull request merged**